### PR TITLE
chore: enhance sentences

### LINKED ISSUE #8107: Improve sentences in user interaction
Please change the sentences as follows:
- Your billing details has been updated. -> Your billing details have been updated.
- zip -> ZIP
   - except when it is related to "zip file". In that case it should be "Zip file".
- An unexpected error occurred. -> Sorry, an unexpected error has occurred. Our developers will fix this.
- Starting time should be lesser than the ending time! -> The start time should be before the end time!
- Ending time should be greater than the starting time! -> The end time should be after the start time!
- Please enter name for track. -> Please enter a name for the track.
- Please enter a Private link. -> Please enter a private link.
- CFS start time should be before than event start time. -> CFS start time should be before event start time.
- Please tell us when your event starts. - When does your event start?
- Please tell us when your event ends. -> When does your event end?
- CFS end time should be before than event start time! -> CFS end time should be before event start time!
- Please give a start time. -> Please enter a start time.
- Please give an end time. -> Please enter an end time.
- Please enter name for microlocation. -> Please enter a name for a microlocation.
- Please add an announcement to enable call for speakers. -> Please add an announcement to enable the call for speakers.
- Please give a name. -> Please enter a name.
- Please give us your tax ID. -> Please enter your tax ID.
- Please give us your organisation name. -> Please enter the name of your organisation.
- Please give us your contact name. -> Please enter a contact name.
- Please give a city. -> Please enter a city.
- Please give a state. -> Please enter a state.
- Your speaker has been saved. -> The speaker has been saved.
- Announce event!! -> Announce event
- YES, accept payment by Purchase on invoice. -> YES, accept payment by purchase on invoice.
- YES, accept payment by Telegraphic Transfer (TT) / Bank Transfer. -> YES, accept payment by telegraphic transfer (TT) / bank transfer.
- Require Billing Information from Ticket Buyer. -> Require billing information from ticket buyer.

What is this?
- {{userThumbnailImg}}and {{followerCount}} others

Screenshot from 2021-11-17 15-48-03

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #8107

### CODE REVIEW

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -182,11 +182,11 @@ export default Component.extend(FormMixin, EventWizardMixin, {
           rules      : [
             {
               type   : 'empty',
-              prompt : this.l10n.t('Please tell us when your event starts')
+              prompt : this.l10n.t('When does your event start?')
             },
             {
               type   : 'date',
-              prompt : this.l10n.t('Please give a valid start date')
+              prompt : this.l10n.t('Please give a valid start date.')
```
Comment (mariobehling) [inline app/components/forms/wizard/basic-details-step.js:189]: Change to "Please enter a valid start date."

File: app/components/forms/wizard/basic-details-step.js [hunk 2]
```diff
@@ -195,15 +195,15 @@ export default Component.extend(FormMixin, EventWizardMixin, {
           rules      : [
             {
               type   : 'empty',
-              prompt : this.l10n.t('Please tell us when your event ends')
+              prompt : this.l10n.t('When does your event end?')
             },
             {
               type   : 'date',
               prompt : this.l10n.t('Please give a valid end date')
```
Comment (mariobehling) [inline app/components/forms/wizard/basic-details-step.js:202]: Change to: Please enter a valid end date.

File: app/components/forms/wizard/basic-details-step.js [hunk 3]
```diff
@@ -276,11 +276,11 @@ export default Component.extend(FormMixin, EventWizardMixin, {
           rules      : [
             {
               type   : 'empty',
-              prompt : this.l10n.t('Please specify how many tickets of this type are available')
+              prompt : this.l10n.t('Please specify how many tickets of this type are available.')
             },
             {
               type   : 'number',
-              prompt : this.l10n.t('Please give a proper quantity for you ticket')
+              prompt : this.l10n.t('Please give a proper quantity for you ticket.')
```
Comment (mariobehling) [inline app/components/forms/wizard/basic-details-step.js:283]: Change to: Please give a proper quantity for your ticket.

**Review by mariobehling — CHANGES_REQUESTED** [review]

**Review by codedsun — COMMENTED** [review]
Follow Commit message guidelines

**Review by mariobehling — APPROVED** [review]

**Pull request merged**